app/javascript/Pages/SingleClass.jsx

```diff
@@ -0,0 +1,220 @@
+          })
+          .catch((error) => {
+            if (error.response.data.error) {
+              alert(error.response.data.error);
+            }
+            setButtonState("available");
+          });
+
+        break;
+
+      default:
+        break;
+    }
+  };
+
+  //button state can be 'available', 'pending', 'accepted' or 'loading'
+  const [buttonState, setButtonState] = useState(
+    props.request.status ? props.request.status : "available"
+  );
+
+  const classes_url_category = () => {
+    return `/classes?category_id=${props.class.skill.category.id}`;
+  };
+
+  const classes_url_skill = () => {
+    return `/classes?category_id=${props.class.skill.category.id}&skill_id=${props.class.skill.id}`;
+  };
+
+  return (
+    <Layout current_user={props.current_user}>
+      <div className="path-wrapper">
+        <Link rel="stylesheet" href="/home" className="path">
+          <img
+            src={require("../images/logo_dark.svg")}
+            alt="logo"
+            className="path-logo"
+          />
+        </Link>
+        <p className="path">&nbsp;/&nbsp;</p>
+        <Link rel="stylesheet" href={classes_url_category()} className="path">
+          {props.class.skill.category.name}
+        </Link>
+        <p className="path">&nbsp;/&nbsp;</p>
+        <Link rel="stylesheet" href={classes_url_skill()} className="path">
+          {props.class.skill.name}
+        </Link>
+        <p className="path">&nbsp;/&nbsp;</p>
+        <Link rel="stylesheet" href="" className="path">
+          {props.class.teacher.name}
+        </Link>
+      </div>
+      <div
+        className="class-header"
+        style={{ "--colorCategory": `#${props.class.skill.category.color}` }}
+      >
+        <h1 className="class-header-title">{props.class.title}</h1>
+        <h2 className="class-header-subtitle">{props.class.teacher.name}</h2>
+      </div>
+      <div>
+        <ul className="class-menu">
+          <li
+            className={click == "about" ? "class-btn active" : "class-btn"}
+            onClick={() => handleClick("about")}
+            style={{
+              "--colorCategory": `#${props.class.skill.category.color}`,
+            }}
+          >
+            About
+          </li>
+          <li
+            className={click == "review" ? "class-btn active" : "class-btn"}
+            onClick={() => handleClick("review")}
+            style={{
+              "--colorCategory": `#${props.class.skill.category.color}`,
+            }}
+          >
+            Review
+          </li>
+          <li
+            className={click == "comments" ? "class-btn active" : "class-btn"}
+            onClick={() => handleClick("comments")}
+            style={{
+              "--colorCategory": `#${props.class.skill.category.color}`,
+            }}
+          >
+            Comments
+          </li>
+        </ul>
+      </div>
+
+      <div className={click == "about" ? "class-about" : "class-about off"}>
```

Comment:
**suggestion (blocking)**: instead of repeating the same class twice you can do `className={"class-about " + (click == "about" ? "" : "off")}` (mind the space after the first class)
(also fix this for the other repeated classes)